app/services/support_interface/candidate_journey_tracker.rb

```diff
@@ -48,19 +48,19 @@ def submitted
     end
 
     def completed_reference_1_requested_at
-      received_references_timings.dig(0, :requested_at)
+      received_references[0]&.requested_at
```

Comment:
I was going to order this like before we previously were and then thought, does it actually matter? Happy to change this there's a reason there should be.